sources/packages/web/src/components/common/sin/AddExpiryDate.vue

```diff
@@ -96,6 +96,7 @@ export default defineComponent({
       formModel,
       checkNotesLengthRule,
       checkStringDateFormatRule,
+      DialogTypes,
```

Comment:
where is this returned DialogTypes used in the template?

Reply:
It is not used, it can be removed. It is an old property added prior to Figma patterns being established for modal. Not sure if @ann-aot will be taking care of it as part of her PR.

Reply:
it was used for a property of `modal-dialog-base`. i removed the property from the component itself as we mo longer ned that